coalib/parsing/Globbing.py

```diff
@@ -217,7 +217,7 @@ def fnmatch(name, globs):
     globs = (globs,) if isinstance(globs, str) else tuple(globs)
```

Comment:
I think the commit message needs to be changed since you're not removing the if statement anymore.

Also "Return False" instead of "Don't return true"

Reply:
done